solr/content.md

```diff
@@ -0,0 +1,63 @@
+# What is Solr?
+Solr is highly reliable, scalable and fault tolerant, providing distributed indexing, replication and load-balanced querying, automated failover and recovery, centralized configuration and more. Solr powers the search and navigation features of many of the world's largest internet sites.
+
+Learn more on [Apache Solr homepage](http://lucene.apache.org/solr/) and in the [Apache Solr Reference Guide](https://www.apache.org/dyn/closer.cgi/lucene/solr/ref-guide/).
+
+> [wikipedia.org/wiki/Apache_Solr](https://en.wikipedia.org/wiki/Apache_Solr)
+
+%%LOGO%%
+
+# How to use this Docker image
+
+To run a single Solr server:
+
+    SOLR_CONTAINER=$(docker run -d -p 8983:8983 -t solr)
```

Comment:
It might be easier for users to understand by using something like `--name some-solr` instead of capturing the container id.  Also, feel free to used fenced code blocks with language when needed.